pkg/kube/inject/webhook.go

```diff
@@ -1305,3 +1383,52 @@ func handleError(message string) {
 	log.Errorf(message)
 	totalFailedInjections.Increment()
 }
+
+// modifies the pod to ensure that the proxy runs with the given proxyUID instead of 1337
+func replaceProxyRunAsUserID(pod *corev1.Pod, proxyUID *int64) {
```

Comment:
usage of this func should not be required anymore as all injection templates now have the `ProxyUID` and `ProxyGID` variables which are inserted during injection